apps/site/assets/ts/schedule/components/line-diagram/LineDiagram.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { ReactElement, useState } from "react";
 import { useFetch } from "react-async";
 import { useInterval } from "use-interval";
+
```

Comment:
❓

Reply:
we introduced `useInterval` in the visibility logging commit, then also used it in the polling code itself, so I had to put the `import { useInterval }` back in...along with a little extra whitespace apparently.